src/metatrain/experimental/pet/default-hypers.yaml

```
@@ -49,5 +54,4 @@ training:
   DO_GRADIENT_CLIPPING: False
   GRADIENT_CLIPPING_MAX_NORM: null # must be overwritten if DO_GRADIENT_CLIPPING is True
   USE_SHIFT_AGNOSTIC_LOSS: False # only used when fitting general target. Primary use case: EDOS
-  ENERGIES_LOSS: per_structure # or per_atom
-  BALANCED_DATA_LOADER: False
+  ENERGIES_LOSS: per_structure # per_structure or per_atom
```

just a single line in the end.

```suggestion
  ENERGIES_LOSS: per_structure # per_structure or per_atom
  
```